ci/docker_ci.sh

```diff
@@ -118,9 +119,8 @@ fi
 # This prefix is altered for the private security images on setec builds.
 DOCKER_IMAGE_PREFIX="${DOCKER_IMAGE_PREFIX:-envoyproxy/envoy}"
 
-
 if is_windows; then
-  BUILD_TYPES=("-windows")
+  BUILD_TYPES=("-windows" "-windows-debug")
```

Comment:
Question, does this cause the linker to run twice, once for windows and once for windows-debug? The issue is that the resulting .pdb file wouldn't be compatible with the resulting .exe file generally distributed to the public. There must be only one linked .exe, and the .pdb captured only for those who care to dive into crash dump files.

Reply:
The docker image build grabs the build output tarball from the windows pipeline run it is in (the previous `windows.release` step) so it should have the `envoy.exe` and `envoy.pdb` file from that one `opt` build run